app/src/main/java/explorer/city/com/cityexplorer/view/adapter/CategoryAdapter.kt

```diff
@@ -0,0 +1,35 @@
+package explorer.city.com.cityexplorer.view.adapter
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import explorer.city.com.cityexplorer.R
+import explorer.city.com.cityexplorer.model.ScoreCategory
+import explorer.city.com.cityexplorer.view.adapter.viewHolder.CategoryViewHolder
+import explorer.city.com.cityexplorer.view.listener.OnItemClickListener
+
+class CategoryAdapter: RecyclerView.Adapter<CategoryViewHolder>() {
+
+    private val categories by lazy { mutableListOf<ScoreCategory>() }
+
+    fun addItems(items: List<ScoreCategory>?) {
+        if (items == null) return
+        categories.clear()
+        categories.addAll(items)
+        notifyDataSetChanged()
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, p1: Int): CategoryViewHolder {
```

Comment:
Or maybe it's not a position -- either way, rename it to something other than `p1`